apis/apps/v1alpha1/opsrequest_types.go

```diff
@@ -66,7 +66,6 @@ type OpsRequestSpec struct {
 	// +patchStrategy=merge,retainKeys
 	// +listType=map
 	// +listMapKey=componentName
-	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="forbidden to update spec.volumeExpansion"
```

Comment:
does CEL `self` work with list type?

Reply:
if not then check all similar attributes in this spec.

Reply:
type Quantity struct {
	// i is the quantity in int64 scaled form, if d.Dec == nil
	i int64Amount
	// d is the quantity in inf.Dec form if d.Dec != nil
	d infDecAmount
	// s is the generated value of this quantity to avoid recalculation
	s string

	// Change Format at will. See the comment for Canonicalize for
	// more details.
	Format
}
if any update/patch occurs, the Quantity object will update the` i, d, s `together. but create the object only set the `s` now